src/Integrations/Integrations/AMQP/AMQPIntegration.php

```diff
@@ -25,6 +30,35 @@ public function getName()
         return self::NAME;
     }
 
+    public static function largeBaseConvert($numString, $fromBase, $toBase)
```

Comment:
Could you add basic attribution in a comment above, at least? This seems 1:1 copied from https://github.com/credomane/php_baseconvert/blob/master/unfucked_base_convert.php.

Reply:
Oups yep doing it right away, I did add it in the [Laravel Queue tests](https://github.com/DataDog/dd-trace-php/blob/d0e97fbcfd3a1b0802848ee7e4e02fad69d34716/tests/Integrations/Laravel/V8_x/QueueTest.php#L53), but forgot to add it there as well.

Reply:
No, don't, we need to drop that function later and replace it by a proper native wrapper.